CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-## X.Y.Z 2017-XX-YY
+## 11.3.0 2017-09-13
+* Improved support and compatibility with iOS 11
 * Changes and fixes methods on `STPCard`, `STPCardParams`, `STPBankAccount`, and `STPBankAccountParams` to bring card objects more in line with the rest of the API. See MIGRATING for further details.
 * Adds nullability annotations to `STPPaymentMethod` protocol
 * Improves the `[STPAPIResponseDecodable allResponseFields]` by removing all instances of `[NSNull null]` including ones that are nested. See MIGRATING.md.
```

Comment:
There's some other changes that need to go in here, such as:
support for p24 in our bindings
support for direct app-to-app Alipay redirects.
fixing this crash https://github.com/stripe/stripe-ios/pull/786

I usually browse through the "x commits to master since this release" link for the latest release when I'm doing the new proposal btw, to make sure all important user facing PRs are covered by the changelog.

Also not sure if we want to expand on the iOS 11 compatibility at all? Or call out the back button changes?